lib/build.gradle

```diff
@@ -102,6 +102,8 @@ android {
 
   defaultConfig {
     minSdkVersion 14
+    //noinspection ExpiredTargetSdkVersion
+    targetSdkVersion 17
```

Comment:
I'm not super comfortable making these changes in the PR. We would have to check internally if there is any workaround or ramification from this. I think we'll probably need to create a separate test module

Can you add `@Ignore("Fix RTL support for robolectric tests")` To the new tests? We can investigate this internally! Thanks so much for working on that!